### PR TITLE
Update with MS Learn learning path content

### PR DESCRIPTION
**Pull request by scottburmester**
Added a link to a complementary MS Learn learning path which supports the Contoso SmartHotel360 scenario-deploying web apps w/ Azure App Services.  Part of an exercise with Brian Blanchard to provide more specific guidance to our customers with links to supporting learning paths from MS Learn.

### CODE REVIEW

**Comment by PRMerger18** [comment]
@scottburmester : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@scottburmester : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by jborsecnik** [comment]
Note: Must remove en-us from URLs.

File: docs/migrate/azure-best-practices/contoso-migration-rebuild.md [hunk 1]
```diff
@@ -600,3 +600,13 @@ With the migrated resources in Azure, Contoso now needs to fully operationalize
 ## Conclusion
 
 In this article, Contoso rebuilds the SmartHotel360 app in Azure. The on-premises app front-end VM is rebuilt to Azure App Service web apps. The application back end is built using microservices deployed to containers managed by Azure Kubernetes Service (AKS). Contoso enhanced app functionality with a pet photo app.
+
+## Suggested Skills
+
+Microsoft Learn is a new approach to learning. Readiness for the new skills and responsibilities that come with cloud adoption doesn't come easily. Microsoft Learn provides a more rewarding approach to hands-on learning that helps you achieve your goals faster. Earn points and levels, and achieve more!
+Here are a couple of examples of tailored learning paths on Microsoft Learn which aligns with the Contoso SmartHotel360 app in Azure.
```
Comment (JanetCThomas) [inline docs/migrate/azure-best-practices/contoso-migration-rebuild.md]: Change aligns to align.

File: docs/migrate/azure-best-practices/contoso-migration-rebuild.md [hunk 2]
```diff
@@ -600,3 +600,13 @@ With the migrated resources in Azure, Contoso now needs to fully operationalize
...
+
+[Deploy a website to Azure with Azure App Service](https://docs.microsoft.com/en-us/learn/paths/deploy-a-website-with-azure-app-service/):Web apps in Azure allow you to publish and manage your website easily without having to work with the underlying servers, storage, or network assets. Instead, you can focus on your website features and rely on the robust Azure platform to provide secure access to your site.
```
Comment (JanetCThomas) [inline docs/migrate/azure-best-practices/contoso-migration-rebuild.md]: Change: ...https://docs.microsoft.com/en-us/learn/paths/deploy-a-website-with-azure-app-service/):Web...
To:
...https://docs.microsoft.com/learn/paths/deploy-a-website-with-azure-app-service/): Web...

File: docs/migrate/azure-best-practices/contoso-migration-rebuild.md [hunk 3]
```diff
@@ -600,3 +600,13 @@ With the migrated resources in Azure, Contoso now needs to fully operationalize
...
+
+[Process and classify images with the Azure Cognitive Vision Services](https://docs.microsoft.com/en-us/learn/paths/classify-images-with-vision-services/):Azure Functions enable the creation of event driven, compute-on-demand systems that can be triggered by various external events. Learn how to leverage functions to execute server-side logic and build serverless architectures.
```
Comment (JanetCThomas) [inline docs/migrate/azure-best-practices/contoso-migration-rebuild.md]: Change: 
(https://docs.microsoft.com/en-us/learn/paths/classify-images-with-vision-services/):Azure 
To:
(https://docs.microsoft.com/learn/paths/classify-images-with-vision-services/): Azure

**Comment by PRMerger16** [comment]
@scottburmester : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/migrate/azure-best-practices/contoso-migration-rebuild.md (hunk 1 above)
Comment (scottburmester) [reply docs/migrate/azure-best-practices/contoso-migration-rebuild.md]: Completed.

File: docs/migrate/azure-best-practices/contoso-migration-rebuild.md (hunk 2 above)
Comment (scottburmester) [reply docs/migrate/azure-best-practices/contoso-migration-rebuild.md]: Removed en-us/ from URLs

File: docs/migrate/azure-best-practices/contoso-migration-rebuild.md (hunk 3 above)
Comment (scottburmester) [reply docs/migrate/azure-best-practices/contoso-migration-rebuild.md]: Removed en-us/ from URLs.

**Comment by ktoliver** [comment]
@scottburmester Thanks for making the changes.

@JanetCThomas @BrianBlanchard Could you review the updated pull request and indicate whether it's ready to merge? Thank you.

**Comment by PRMerger12** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**